docs/features/workspaces.md

```diff
@@ -76,3 +77,24 @@ Start by creating a workspace name, then click "Add repositories." You'll have t
 3. **Import Repositories**: Paste a list of repositories to add to your workspace.
 
 Once your repositories are added, you'll be able to see them in your repository dashboard. You can also edit your workspace to add or remove repositories anytime.
+
+### Changing Workspaces
+
+Anything you create in a workspace is specific to that workspace. When you switch workspaces, you'll see the repositories and contributors specific to that workspace.
+
+## How to Share a Workspace
+
+You can share your workspace with others by copying the URL and sending it to them. If your workspace is private (a PRO plan feature), only those with access to the workspace will be able to view it.
+
+
+## Team Workspaces
+
+Team workspaces are part of the PRO plan and are designed to help teams collaborate on and understand open source projects. Team workspaces are public by default. You can switch it to private in your settings so only members of the team can view the workspace.
```

Comment:
I think a slight of wording format change here would be better for clarity.

```suggestion
Team workspaces are part of the PRO plan and are designed to help teams collaborate on and understand open source projects. By default, team workspaces are public. However, you can switch it to private in your settings so only the team members can view it.
```

Reply:
I think it's better to start with the subject of the sentence. So Workspaces are private by default is preferred in this instance. But I did adjust structure based on your suggestions.